plugins/homepage/plugin_tests/homepage_test.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2014 Kitware Inc.
```

Comment:
`Copyright 2016 Kitware Inc.`

Although, it is kinda goofy that we have copyright on Python files and no others.  Not sure why that is.